docs/static/openapi.yml

```diff
@@ -7153,624 +7288,24 @@ paths:
       tags:
         - Query
 definitions:
-  zigbeealliance.distributedcomplianceledger.compliance.CertifiedModel:
```

Comment:
Why is it removed?

Reply:
By mistake did't realize some cosmos fields were in between, should be fine now.